go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/cucumber/godog v0.12.1
 	github.com/dell/dell-csi-extensions/podmon v1.0.0
 	github.com/dell/dell-csi-extensions/volumeGroupSnapshot v0.0.0-20211104171507-7c7d0b6e3d35
-	github.com/dell/gocsi v1.3.1
+	github.com/dell/gocsi v1.4.0
```

Comment:
if latest gocsi is in main branch, but not released yet, can you uncomment the replace statement so we build and test with it in pipelines?

Reply:
1.4 is already released. They are going to release 1.4.1 this quarter